shimmer/docs/wallet.rs/docs/_admonitions/_warning_unlock_conditions.md

```diff
@@ -6,6 +6,6 @@ To get outputs with only the [`AddressUnlockCondition`](https://wiki.iota.org/sh
 
 If you are synchronizing outputs with other unlock conditions, you should check the unlock conditions carefully before crediting users any balance.
 
-You can find an example illustrating how to check if an output has only the address unlock condition, where the address belongs to the account in the [Check Unlock Conditions how-to guide](../how_tos/outputs_and_transactions/06_check_unlock_conditions.mdx).
+You can find an example illustrating how to check whether an output has only the address unlock condition and whether this address belongs to the account in the [Check Unlock Conditions how-to guide](https://github.com/iotaledger/iota-sdk/blob/develop/bindings/nodejs/examples/wallet/17-check-unlock-conditions.ts).
```

Comment:
This isn't needed in the shimmer docs. 

```suggestion
You can find an example illustrating how to check whether an output has only the address unlock condition and whether this address belongs to the account in the [Check Unlock Conditions how-to guide](../how_tos/outputs_and_transactions/06_check_unlock_conditions.mdx).
```

Reply:
@Dr-Electron but in the shimmer wiki there isn't the sdk, so how does this all match?

Reply:
The SDK isn't released yet. So thy do we care? On Shimmer you should still use wallet.rs and iota.rs

Reply:
No they should use the SDK, it'll be release by then

Reply:
I got it. But this was just a wrong replacement in the Shimmer part of the wiki instead of next where the SDK docs currently are located 👍